it/common/src/main/java/org/apache/beam/it/common/matchers/package-info.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/** Package for Truth matchers / subjects to have reusable assertions. */
+package org.apache.beam.it.common.matchers;
```

Comment:
Since it is specifically for truth matchers, perhaps `org.apache.beam.it.truthmatchers` or something. It is fine as-is but imagine we had built the framework for Hamcrest and then Truth came out and we wanted to also support it. Separating modules according to the dependencies they pull in is often a good idea.